app/about/us/page.tsx

```diff
@@ -0,0 +1,137 @@
+import React from "react"
+import { Hero } from "@/components/Hero"
+import { TextAnimate } from "@/components/magicui/text-animate"
+import { SectionHeader } from "@/components/ui/section-header"
+import { Card, CardContent, CardWithImage } from "@/components/ui/card"
+
+const people = [
+  {
+    name: "Ashley Lee",
+    title: "Senior Research Software Engineer",
+    href: "/ashley-lee",
```

Comment:
I don't think we'll have links to bios? But we could

Reply:
yea i made it an issue to add later... our current website has this